### ISSUE TITLE
IntegerMultiplier.java missing?

### ISSUE THREAD

**Issue by rvandam** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#197]
NumberIsMultipleOfAnotherNumberVerifier.java consumes IntegerDivider.java properly but could be improved with an analogous implementation of IntegerMultiplier.java.  Abstracting division without abstracting multiplication seems like an abstract failure to sufficiently abstract.  I imagine this request may be divisive but as it stands, there is no way to account for potential future changes such as extending the strategy to accommodate vector math which would require a more robust cross product capability which is currently impossible.

**Comment by Dmitry-Me** [comment]
Should be *an abstract failure to sufficiently `abstractly` abstract*.

**Comment by pauldraper** [comment]
Agreed, that abstracting one without abstracting the other is an failure (however abstract) in abstraction.

**Comment by vendethiel** [comment]
Shouldn't IntegerMultiplier and IntegerDivider inherit an abstract IntegerOperation?

**Comment by jevinskie** [comment]
We should probably add UnaryIntegerOperation and BinaryIntegerOperation under IntegerOperation. Just in case.
